_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -330,30 +333,36 @@ POST /_snapshot/my-repository/2/_restore
 
 Request parameters | Description
 :--- | :---
-`indices` | The indices you want to restore. You can use `,` to create a list of indices, `*` to specify an index pattern, and `-` to exclude certain indices. Don't put spaces between items. Default is all indices.
+`indices` | The indexes you want to restore. You can use `,` to create a list of indexes, `*` to specify an index pattern, and `-` to exclude certain indexes. Don't put spaces between items. Default is all indexes.
```

Comment:
Entire list: Should the defaults be in tic marks?

Reply:
Yes, looks like we add tic marks to the default values.